src/js/worklets/communities.js

```diff
@@ -45,19 +70,19 @@ export function useLogoStyles({
   });
 }
 
-export function useSheetStyles({ scrollAmount, expandHeaderThreshold, sheetDisplacementThreshold }) {
+export function useSheetStyles({ initialState, scrollAmount, collapseThreshold, sheetDisplacementThreshold }) {
   return useAnimatedStyle(() => {
-    const firstDisplacement = scrollAmount.value < expandHeaderThreshold;
-    if (firstDisplacement) {
+    const isFirstDisplacement = initialState.value === 'expanded' || scrollAmount.value < collapseThreshold.value;
```

Comment:
Tiny comment: Maybe we can create a helper function for computing `isFirstDisplacement`?